lib/js-api/src/api/content-rest-api/docs/SettingsApi.md

```diff
@@ -0,0 +1,129 @@
+# SitesApi
+
+All URIs are relative to *https://localhost/alfresco/api/-default-/public/alfresco/versions/1/*
+
+| Method                                                 | HTTP request                   | Description                                                                         |
+|--------------------------------------------------------|--------------------------------|-------------------------------------------------------------------------------------|
+| [getSavedExtensionState](#getSavedExtensionState)      | **GET** /settings/{instanceId} | Gets the extension configuration saved for a running instance of ADW on the backend |
+| [publishExtensionConfig](#publishExtensionComposition) | **PUT** /settings/{instanceId} | Saves an extension configuration for a running instance of ADW on the backend       |
+
+## getSavedExtensionState
+
+Gets the extension configuration saved on the backend
+
+> this endpoint is available in <GET VERSION INFO HERE>**Alfresco 7.0.0** and newer versions.
+
+**Parameters**
+
+| Name           | Type   | Description                                  | Notes |
+|----------------|--------|----------------------------------------------|-------|
+| **instanceId** | string | The identifier of a running instance of ADW. |       |
+
+**Return type**: [ExtensionCompositionEntry](#ExtensionCompositionEntry)
+
+## publishExtensionComposition
+
+Saves an extension configuration on the backend
+
+**Parameters**
+
+| Name                | Type                                | Description                                  |
+|---------------------|-------------------------------------|----------------------------------------------|
+| **instanceId**      | string                              | The identifier of a running instance of ADW. |
```

Comment:
we should not mention ADW in documentation for js-api because js-api is reusable library so should know nothing about ADW